blend2bam/blend2gltf/blender28_script.py

```diff
@@ -158,6 +165,7 @@ def export_gltf(settings, src, dst):
     add_actions_to_nla()
 
     prepare_meshes()
+    remove_hide_render()
```

Comment:
We might as well do this first to avoid performing operations (e.g., `prepare_meshes()`) on objects we're just going to remove anyways.

Reply:
In my particular use-case (a heightmap), I still wanted the particle effects to be applied before it is removed.

Reply:
It could go before `prepare_meshes() ` but after `common.make_particles_real()`